src/ui/Pricing/Page/PromoSanTokenBanner.svelte

```diff
@@ -70,5 +103,9 @@
     p {
       margin: 0 0 32px;
     }
+
+    .banner-phone {
```

Comment:
Class is redundant, since selector is inside `phone`/`phone-xs` parent
```suggestion
    img {
```